app.js

```diff
@@ -1,9 +1,85 @@
+const axios = require("axios");
 const express = require("express");
+const { response } = require("express");
+const fs = require("fs");
+const hasha = require("hasha");
+
 const app = express();
 const port = 3000;
 
 app.get("/", (req, res) => res.send("Hello World!"));
 
+app.get("/:user/:repo/compile", (req, res) => {
+  const user = req.params.user;
+  const repo = req.params.repo;
+  getReadme(user, repo).then(() => {
+    res.send(`Successfully Compiled Docs! View them at /${user}/${repo}`);
+  });
+});
+
+// Gets readme contents and generates /docs/<files> with docsify enabled to render
+async function getReadme(user, repo) {
+  const readme = await axios.get(
+    `https://api.github.com/repos/${user}/${repo}/readme`
+  );
+  const readmeUrl = readme.data.download_url;
+  const readmeData = await axios.get(readmeUrl);
+
+  if (!fs.existsSync("./docs/")) fs.mkdirSync("./docs/");
+
+  if (!fs.existsSync(`./docs/${user}-${repo}/`))
+    fs.mkdirSync(`./docs/${user}-${repo}/`);
```

Comment:
This style of shorthand is ESPECIALLY not recommended. We don't want this kind of whitespace in JS.